src/scripts/main.js

```diff
@@ -1,3 +1,269 @@
 'use strict';
 
-// write your code here
+const startButton = document.querySelector('.button');
+const gameField = document.querySelector('.game-field');
+const gameScore = document.querySelector('.game-score');
+const cells = document.querySelectorAll('.field-cell');
```

Comment:
Just comment: instead of use spread in few places you can make cell prepared before
```suggestion
const cells = [...document.querySelectorAll('.field-cell')];
```

Reply:
thank you! so simple but really useful :)